sbt-launcher/src/test/scala/coursier/sbtlauncher/ScriptedTests.scala

```diff
@@ -20,20 +20,11 @@ object ScriptedTests extends TestSuite {
         )
       }
 
-      "sbt 1.3.0-M2" - {
-        val dir = Paths.get("tests/scripted-check-1.3.0-M2")
```

Comment:
Let's keep this test. It should keep passing. If you need to add tests, add them along the existing ones.